File: Odd Numbers.c
#include<stdio.h>

void main()
{
int a,b,i;
printf("\n\t***Odd Numbers***");
printf("\n\t=================");
printf("\nEnter starting number: ");
scanf("%d", &a);
printf("\nEnter ending number: ");
scanf("%d", &b);
printf("\nODD NUMBERS");
printf("\n\nFor Loop\n");
for(i=a; i<=b; i++)
{
    if(i%2==1)
        printf("%d\t", i);
}
printf("\n\nWhile Loop\n");
i=a;
while(i<=b)
{
    if(i%2==1)
        printf("%d\t", i);
    i++;
}
printf("\n\nDo While Loop\n");
i=a;
do
{
    if(i%2==1)
        printf("%d\t", i);
    i++;
}while(i<=b);
}

File: Odd_Numbers.java
import java.util.*;

public class Odd_Numbers{
    public static void main(String[] args){
        int abc[] = {1,0,25,29,7,654,99,4,595,132,789,123};
        int j=1;
        int odd,i;
        for(i=0;i<abc.length;i++)
        {
            if (abc[i]%2==1)
            {
                odd = abc[i];
                System.out.println("Odd Number "+j+" = "+odd);
                j++;
            } 
        }
    }
}